code/framework/src/networking/network_client.cpp

```diff
@@ -101,55 +101,55 @@ namespace Framework::Networking {
             }
             _state = CONNECTED;
             return true;
-        };
+        }
 
         case ID_NO_FREE_INCOMING_CONNECTIONS: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::NO_FREE_SLOT);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
 
         case ID_DISCONNECTION_NOTIFICATION: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::GRACEFUL_SHUTDOWN);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
 
         case ID_CONNECTION_LOST: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::LOST);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
 
         case ID_CONNECTION_ATTEMPT_FAILED: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::FAILED);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
 
         case ID_INVALID_PASSWORD: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::INVALID_PASSWORD);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
 
         case ID_CONNECTION_BANNED: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::BANNED);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
         }
```

Comment:
two brackets at the same ident?

Reply:
yes, notice switch case is pushed down, this is fine.